deploy/kubernetes/operator/pkg/.gitkeep

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
If operator directory have files, could we remove this file?

Reply:
Yes